CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Labone Python API Changelog
 
+## Version 3.2.0
+* `subscribe` accepts keyword arguments, which are forwarded to the data-server.
+  This allows to configure the subscription to the data-server.
+  Note that as of LabOne 24.10, no node supports yet subscription configuration.
+
 ## Version 3.1.2
```

Comment:
suggestion: If we do not release a separate version 3.1.2 I would just merge it with 3.2.0